exercises/wordle_application.livemd

```diff
@@ -40,18 +40,18 @@ If a letter is in the word, but in the incorrect position, it will highlight yel
 You should be able to start the application using `iex -S mix`
 
 ```
-$ iex -S mix
+iex -S mix
```

Comment:
Why remove the `$`? In general we've used `$` for command line instructions.

Reply:
I remembered it was removed to follow the convention of previous command that was without a `$`:

https://github.com/DockYard-Academy/beta_curriculum/pull/284/files#diff-37485c17a43492d49b4ccd31a95bb0f7a9fd014a24f725c3c1598fe0037f6acfR28

Does Livebook supports copy-and-page for code block like GitHub did for Markdown file?
If not, we'll stick to the convention of having `$` for all command line instruction.

Reply:
Ooooh that's a really cool idea. I bet I could make that with a smart cell.

To avoid blocking this PR I'm going to merge this in for now and then either change it or see if I can make a smart cell with the copy functionality.

Alternatively this might be a good issue for livebook 🤔